docs/source/multi_gpu.rst

```diff
@@ -260,7 +260,7 @@ Distributed Data Parallel
 
 3. Each process inits the model.
 
-.. note:: Make sure to set the random seed so that each model initializes with the same weights.
+.. note:: Make sure to set the random seed before the instantiation of a `Trainer()` so that each model initializes with the same weights.
```

Comment:
```suggestion
.. note:: Make sure to set the random seed before the instantiation of a ``Trainer()`` so that each model initializes with the same weights.
```